openedx-unsupported/configuration
# Populate ENABLE_OAUTH2_PROVIDER in [cl]ms.env.yaml.

**Description**: This change adds a variable: `EDXAPP_ENABLE_OAUTH2_PROVIDER`.

This allows the `FEATURES` list to be populated with `ENABLE_OAUTH2_PROVIDER` set to the value of `EDXAPP_ENABLE_OAUTH2_PROVIDER`.

Previously, it was necessary to edit `/edx/etc/edxapp/[cl]ms.env.yaml` and add/set `ENABLE_OAUTH2_PROVIDER` manually. 

Ansible users needing to control `ENABLE_OAUTH2_PROVIDER` can now set `EDXAPP_ENABLE_OAUTH2_PROVIDER` to "true" to override the default, which is "false".

**Dependencies**: None

**Screenshots**: Not applicable.

**Sandbox URL**: Not applicable.

**Partner information**: Not applicable.

**Testing instructions**:
1. Run `sudo /edx/bin/ansible-playbook -c local -i "localhost," edx_sandbox.yml` and confirm successfule completion.
2. Run `grep OAUTH2 /edx/etc/edxapp/*yaml`; the result should be

`/edx/etc/edxapp/cms.env.yaml:    ENABLE_OAUTH2_PROVIDER: false`
`/edx/etc/edxapp/lms.env.yaml:    ENABLE_OAUTH2_PROVIDER: false`
(Prior to this change, grep would return no output.)

**Reviewers**
@omarkhan, TBD
